### PR TITLE
docs: document how to run in docker

### PR DESCRIPTION
**Pull request by marcoieni**
Trying to save hours of debugging to other people 😅 
After many attempts I got cargo-http-registry working in https://github.com/MarcoIeni/release-plz/pull/421 🥳

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,36 @@ setting, such as on a single computer or local home network. The reason
...
+    image: ghcr.io/d-e-s-o/cargo-http-registry:latest
+    container_name: cargo-registry
+    restart: always
+    # Arguments:
+    # - Directory where the registry will store its data.
+    #   Cargo-http-registry creates the directory if it does not exist.
+    # - Server address.
+    #   Note that 127.0.0.1 doesn't work in GitHub actions.
+    #   Use 0.0.0.0 instead.
+    command: /tmp/test-registry -a 0.0.0.0:35504
```
Comment (d-e-s-o) [inline README.md:135]: Let's use long options in code snippets?
```suggestion
    command: /tmp/test-registry --addr 0.0.0.0:35504
```

**Review by d-e-s-o — APPROVED** [review]
Very nice, thanks! Left one suggestion. Is there a reasoning behind the port range or is it just the first unassigned random port that popped up?

**Comment by marcoieni** [comment]
Just a random one. No reasons.
It's the one mentioned in readme +1.

**Comment by d-e-s-o** [comment]
Closed via https://github.com/d-e-s-o/cargo-http-registry/commit/21cd54824d82e19ca06b07289628eb9496a1e99a

**Pull request closed without merging**